redis/asyncio/connection.py

```diff
@@ -347,6 +347,15 @@ async def on_connect(self) -> None:
             if str_if_bytes(await self.read_response()) != "OK":
                 raise ConnectionError("Error setting client name")
 
+        try:
+            # set the library name and version
+            await self.send_command("CLIENT", "SETINFO", "LIB-NAME", "redis-py")
+            await self.read_response()
```

Comment:
Any reason to not pipeline this?,,, and just not read the response? From the docs:
`Client libraries are expected to pipeline `